api/client/stats_helpers.go

```diff
@@ -24,6 +24,7 @@ type containerStats struct {
 	NetworkTx        float64
 	BlockRead        float64
 	BlockWrite       float64
+	PidsCurrent      uint64
```

Comment:
Oh I see yeah that was definitely lost in a rebase :(

On Tuesday, March 8, 2016, Tõnis Tiigi <notifications@github.com> wrote:

> In api/client/stats_helpers.go
> <https://github.com/docker/docker/pull/18697#discussion_r55465554>:
>
> > @@ -24,6 +24,7 @@ type containerStats struct {
> >  	NetworkTx        float64
> >  	BlockRead        float64
> >  	BlockWrite       float64
> > +	PidsCurrent      uint64
>
> That's setting cgroup value to the api type on daemon side. PidsCurrent
> is client side value used for printing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18697/files#r55465554>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>